examples/with-mobx-state-tree/pages/_app.js

```diff
@@ -4,29 +4,33 @@ import { getSnapshot } from 'mobx-state-tree'
 import App from 'next/app'
 import { initializeStore } from '../stores/store'
 
-export default class MyApp extends App {
-  static async getInitialProps({ Component, ctx }) {
-    //
-    // Use getInitialProps as a step in the lifecycle when
-    // we can initialize our store
-    //
-    const isServer = typeof window === 'undefined'
-    const store = initializeStore(isServer)
-    //
-    // Check whether the page being rendered by the App has a
-    // static getInitialProps method and if so call it
-    //
-    let pageProps = {}
-    if (Component.getInitialProps) {
-      pageProps = await Component.getInitialProps(ctx)
-    }
-    return {
+export async function getStaticProps({ Component, ctx }) {
+  //
+  // Use getStaticProps as a step in the lifecycle when
+  // we can initialize our store
+  //
+  const isServer = typeof window === 'undefined'
+  const store = initializeStore(isServer)
```

Comment:
I don't think initializing the store here in `getStaticProps` provides any value